app/views/hvd/cash_payment_first_date.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import include._
+@import forms2._
+
+@(f: forms.Form2[_], edit: Boolean)(implicit request: Request[_],m:Messages)
+
+@header = {
+    @components.back_link()
+    @errorSummary(f)
+    @heading("hvd.cash.payment.date.title", "summary.hvd")
+}
+
+@main(
+    title = Messages("hvd.cash.payment.date.title") + " - " + Messages("summary.hvd"),
+    heading = header
+)  {
+
+    @form(f, controllers.hvd.routes.CashPaymentFirstDateController.post(edit)) {
+
+        <div class="form-group">
+
+        @fieldset(
+            f = f("paymentDate-fieldset"),
+            legend = "hvd.cash.payment.date.title",
+            legendHidden = true,
+            panel = false,
+            classes = Seq("inline")
+        ) {
+            @errorGroup(
+                field = f("paymentDate")
+            ) {
+                @date(
+                    f = f,
+                    p = "paymentDate",
+                    labelText = "hvd.cash.payment.date.title",
+                    hintText = "hvd.cash.payment.date.example",
+                    example = false
+                )
+            }
+        }
+
+        </div>
```

Comment:
Found a bug in the code in which the question is duplicated and also the error message is displayed three times when no date is entered. 

Suggest replacing lines 35 through 57 with the following;

       ```
@date(
          f = f,
          p = "paymentDate",
          labelText = "hvd.cash.payment.date.title",
          hintText = "hvd.cash.payment.date.example",
          legendHidden = true,
          example = false
       )
```